templates/terraform/examples/app_engine_flexible_app_version.tf.erb

```diff
@@ -0,0 +1,84 @@
+resource "google_app_engine_flexible_app_version" "<%= ctx[:primary_resource_id] %>" {
+  version_id = "v1"
+  service    = "<%= ctx[:vars]['service_name'] %>"
+  runtime    = "nodejs"
+
+  entrypoint {
+    shell = "node ./app.js"
+  }
+
+  deployment {
+    zip {
+      source_url = "https://storage.googleapis.com/${google_storage_bucket.bucket.name}/${google_storage_bucket_object.object.name}"
+    }
+  }
+
+  liveness_check {
+    path = "/"
+  }
+
+  readiness_check {
+    path = "/"
+  }
+
+  env_variables = {
+    port = "8080"
+  }
+
+  automatic_scaling {
+    cool_down_period = "120s"
+    cpu_utilization {
+      target_utilization = 0.5
+    }
+  }
+
+
+  delete_service_on_destroy = true
+}
+
+resource "google_app_engine_flexible_app_version" "myflexapp_v2" {
```

Comment:
nit: I have a slight preference for these to be 2 different example blocks instead of 2 app_versions in a single example. If possible I like to have each example be a single use case so that the example can have a title block like "NodeJs Example with scaling" and a user doesn't have to parse both whole resource blocks to understand the difference between the two. As pointed out by a user [aws s3 bucket](https://www.terraform.io/docs/providers/aws/r/s3_bucket.html) does a better job of this than many of our resources.

Reply:
I'm totally fine with that, I copied this from the standard app (again), and it appears both versions are the same.  I'm guessing the example of it was just having 2 versions to the same service?  But if that doesn't seem like a great thing to show in an example, I'm happy to remove v2.

Reply:
The only difference I saw was `noop_on_destroy = true` vs `delete_service_on_destroy = true` which doesn't seem like it's worth all of the wasted real estate on the docs page.